modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheProcessor.java

```diff
@@ -17,37 +17,9 @@
 
 package org.apache.ignite.internal.processors.cache;
 
-import java.io.Serializable;
```

Comment:
Please change settings in your IDE that affect on how classes are imported - either the whole package or a particular class.

Recently Ignite community switched from 'package' based imports to a per class based import. So you shouldn't remove all these imports.

To make things simpler please set {ignite}/idea/ignite_codeStyle.xml that will setup everything for you.